app/controllers/api/v1/customers_controller.rb

```diff
@@ -110,6 +112,16 @@ def include_options
 
         { include: fields.map(&:to_s) }
       end
+
+      def customer_already_exists?(customer)
+        Customer.exists?(email: customer.email)
+      end
+
+      def update_created_manually_flag(customer)
+        return if customer.email.blank?
+
+        Customer.where(email: customer.email).update_all(created_manually: true)
+      end
```

Comment:
This is a repeat from the other controller. Maybe the Customer model itself could have a method called `find_or_new`.